src/formatDate.js

```diff
@@ -9,6 +9,34 @@
  */
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const seperator = date.split(fromFormat[3]);
```

Comment:
The variable name 'seperator' is a misspelling of 'separator'. It's important to use correct spelling for variable names to avoid confusion and maintain code quality.